R/hugo.R

```diff
@@ -344,6 +344,8 @@ install_theme = function(
     }
     # delete the .Rprofile if exists, since it's unlikely to be useful
     unlink(file.path(zipdir, '.Rprofile'))
+    # delete the .github folder if exists, since it's unlikely to be useful too
+    unlink(file.path(zipdir, '.github'), recursive = TRUE)
```

Comment:
```suggestion
    # delete the .Rprofile and .github folder if they exist, since they are unlikely to be useful
    unlink(file.path(zipdir, c('.Rprofile', '.github')), recursive = TRUE)
```